packages/vscode-extension/src/controls/QuickStart.tsx

```diff
@@ -246,17 +246,17 @@ export default class QuickStart extends React.Component<any, any> {
               stepCount++;
               return (
                 <GetStartedAction
-                  title={`Build your first Teams app from a sample`}
+                  title={`Build your first Teams app`}
                   content={[
-                    "Explore our sample apps to quickly get started with concepts and code examples.",
+                    "Build a Teams app from the scratch or explore our samples to help you quickly get started with the basic Teams app concepts and code structures.",
                     <br />,
                     <br />,
-                    "You can also create a new project from scratch.",
+                    "You can also create a new project from samples.",
                   ]}
-                  actionText="View all Samples"
```

Comment:
You need to move the logics of mark step as done from viewAllSamples to createNewProject.